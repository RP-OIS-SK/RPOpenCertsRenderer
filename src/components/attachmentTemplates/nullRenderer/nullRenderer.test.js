import React from "react";
import renderer from "react-test-renderer";
import NullRenderer from "./index";

it("matches snapshot", () => {
  const attachment = { type: "application/epub+zip" };
  const component = renderer.create(<NullRenderer attachment={attachment} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
