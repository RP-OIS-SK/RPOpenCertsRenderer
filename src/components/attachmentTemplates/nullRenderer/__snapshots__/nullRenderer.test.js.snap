// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <h2>
    Rendering of this type of attachment is not supported.
  </h2>
  <p>
    Please check your mimetype: 
    application/epub+zip
  </p>
</div>
`;
